src/veh_interact.cpp

```diff
@@ -1269,7 +1269,7 @@ void veh_interact::do_repair()
         bool would_prevent_flying = veh->would_repair_prevent_flyable( pt, player_character );
         if( would_prevent_flying &&
             !player_character.has_proficiency( proficiency_prof_aircraft_mechanic ) ) {
-            nmsg += _( "\n<color_yellow>You require the Airframe and Powerplant Mechanics proficiency to repair this part safely!</color>\n\n" );
+            nmsg += _( "\n<color_yellow>You require the \"Airframe and Powerplant Mechanic\" proficiency to repair this part safely!</color>\n\n" );
```

Comment:
```suggestion
            nmsg += string_format(
                        _( "\n<color_yellow>You require the \"%s\" proficiency to repair this part safely!</color>\n\n" ),
                        proficiency_prof_aircraft_mechanic->name() );
```
I think it would be better to pull the name directly from the JSON definition